packages/api/portfolioDrafts/application/createApplicationStep.test.ts

```diff
@@ -45,7 +45,7 @@ const validRequest: APIGatewayProxyEvent = {
   pathParameters: { portfolioDraftId: uuidv4() },
 } as any;
 
-describe("Handle service level error", function () {
+describe("Handle service level error", () => {
```

Comment:
Glad to see the consistency here. Thank you!

Reply:
Did a global replace for this.